Users/Install.rst

```diff
@@ -83,10 +83,80 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environment and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your project
+directory.
+
+    On **Unix based** systems (OS X and Linux), you can find the user base
+    binary directory for your pipenv by running
+
+    ::
+
+        $ python -m site --user-base
```

Comment:
put an example on what might be the output here

Reply:
Sure. I'm working on above two issues.